protocol/transport/filters_manager.go

```diff
@@ -168,7 +168,8 @@ func (f *FiltersManager) InitCommunityFilters(communityFiltersToInitialize []Com
 		// TODO: requests to join / cancels are currently being sent into the default waku topic.
 		// They must be sent into an specific non protected shard
 		for _, pubsubTopic := range topics {
-			identityStr := PublicKeyToStr(&cf.PrivKey.PublicKey)
+			pk := &cf.PrivKey.PublicKey
```

Comment:
So here, given that we're taking an address of loop variable `cf`, this `pk` param would always point to the same var, correct?

Reply:
yeah!
Earlier I updated golangci-lint on my machine and when I executed `make lint` saw that warning!